accounts/forms.py

```diff
@@ -20,11 +20,27 @@ class Meta:
 class UserUpdateForm(forms.ModelForm):
     """created form for update user profile"""
 
-    phone_no = forms.RegexField(min_length=10, max_length=10, regex=r'^\d+$',
-                                error_messages={'invalid': 'Enter a valid integer.'})
-    email = forms.RegexField(regex= '^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
-                             ,
-                             error_messages={'invalid': 'Enter a valid email address.'})
+    first_name = forms.RegexField(max_length=30, regex='^[a-zA-Z._%+-]+$',
```

Comment:
Keep all regex in single constant file instead of define in the form field directly

Reply:
Done